x-pack/metricbeat/module/iis/application_pool/reader.go

```diff
@@ -9,40 +9,41 @@ package application_pool
 import (
 	"strings"
 
+	"github.com/elastic/beats/v7/libbeat/common"
+	"github.com/elastic/beats/v7/metricbeat/helper/windows/pdh"
```

Comment:
nit: please fix import's order

Reply:
ran fmt so that should be the order

Reply:
I don't think that `mage fmt` touches the order, I just made a small local test and it seems that doesn't. You can verify that by putting them in order and running again `fmt`.
Not sure how this change occurred and preserving order is not crucial but it's a more convention we follow.